source/imaer-util/src/main/java/nl/overheid/aerius/util/NcaOptionsMetadataUtil.java

```diff
@@ -0,0 +1,212 @@
+/*
+ * Crown copyright
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see http://www.gnu.org/licenses/.
+ */
+package nl.overheid.aerius.util;
+
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.ADMS_COMPLEX_TERRAIN_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.ADMS_PLUME_DEPLETION_NH3_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.ADMS_PLUME_DEPLETION_NOX_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.MIN_MONIN_OBUKHOV_LENGTH_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.PRIESTLEY_TAYLOR_PARAMETER_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.ROUGHNESS_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.SPATIALLY_VARYING_ROUGHNESS_DEFAULT;
+import static nl.overheid.aerius.shared.domain.v2.characteristics.adms.ADMSLimits.SURFACE_ALBEDO_DEFAULT;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.LIST_OPTION_SEPARATOR;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.OPTION_KEY_SPLIT;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.addBooleanValue;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.addIntValue;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.addValue;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.getOrDefault;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.isOrDefault;
+import static nl.overheid.aerius.util.OptionsMetadataUtil.parseListOption;
```

Comment:
Guess this works, but it's a bit unusual? Also makes the classes a bit cyclical dependant, as `OptionsMetadataUtil` uses `NcaOptionsMetadataUtil`, which uses `OptionsMetadataUtil`?

Reply:
Agreed. I thought about that too, but I should have followed up on that. So added new base class for generic methods.